src/NuGet.Core/NuGet.Packaging/Signing/Verification/VerificationAllowListObject.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Packaging.Signing
+{
+    public abstract class VerificationAllowListObject
```

Comment:
I would drop `Object` from the name and use something more meaningful. `Entry` would be fine also.